src/validation.h

```diff
@@ -1047,6 +1047,12 @@ class ChainstateManager
     //!          that a background validation chainstate is also in use.
     bool IsSnapshotActive() const;
 
+    //! @returns true if the chainstate is the background chainstate
+    bool IsBackgroundChainstate(const Chainstate *chainstate) const EXCLUSIVE_LOCKS_REQUIRED(::cs_main)
+    {
+        return chainstate == m_ibd_chainstate.get();
+    }
```

Comment:
https://github.com/bitcoin/bitcoin/pull/27746/commits/55e27ee995944d188d7453c6e78a6538f6739d64

Maybe not necessary in lieu of `this != &ActiveChainstate()`? Not hurting anything though.

Reply:
Agreed, gone now.